apps/contact-service/src/routes/clubs/member/getClubContacts.ts

```diff
@@ -0,0 +1,57 @@
+import {NotFoundError} from '@vexl-next/domain/src/general/commonErrors'
+import {
+  GetClubContactsErrors,
+  UserNotClubMemberError,
+} from '@vexl-next/rest-api/src/services/contact/contracts'
+import {GetClubContactsEndpoint} from '@vexl-next/rest-api/src/services/contact/specification'
+import makeEndpointEffect from '@vexl-next/server-utils/src/makeEndpointEffect'
+import {validateChallengeInBody} from '@vexl-next/server-utils/src/services/challenge/utils/validateChallengeInBody'
+import {Effect} from 'effect'
+import {Handler} from 'effect-http'
+import {ClubMembersDbService} from '../../../db/ClubMemberDbService'
+import {ClubsDbService} from '../../../db/ClubsDbService'
+
+export const getClubContacts = Handler.make(GetClubContactsEndpoint, (req, _) =>
+  makeEndpointEffect(
+    Effect.gen(function* (_) {
+      yield* _(validateChallengeInBody(req.body))
+
+      const clubsDb = yield* _(ClubsDbService)
+      const clubMembersDb = yield* _(ClubMembersDbService)
+
+      const club = yield* _(
+        clubsDb.findClubByUuid({uuid: req.body.clubUuid}),
+        Effect.flatten,
+        Effect.catchTag('NoSuchElementException', (e) => new NotFoundError())
```

Comment:
I think it's a good practice to use message. Like this: 

```suggestion
        Effect.catchTag('NoSuchElementException', (e) => new NotFoundError({message: 'Club not found'}))
```